### PR TITLE
Delete import from example

### PR DESCRIPTION
**Pull request by nohkwak**

### CODE REVIEW

File: ethers-ext/example/transactions/Basic_20_TxTypeAccountUpdate.js [hunk 1]
```diff
@@ -27,7 +26,7 @@ async function main() {
         from: senderAddr,
         key: {
             type: 0x02, 
-            key: crypto.computePublicKey(senderNewPriv, true),
+            key: wallet.computePublicKey(senderNewPriv, true),
```
Comment (blukat29) [inline ethers-ext/example/transactions/Basic_20_TxTypeAccountUpdate.js:29]: There's `ethers.utils.computePublicKey`. https://docs.ethers.org/v5/api/utils/signing-key/
It's awkward for a Wallet to have this method.

File: ethers-ext/example/transactions/Basic_20_TxTypeAccountUpdate.js [hunk 2]
```diff
@@ -1,5 +1,4 @@
 const ethers = require("ethers");
-import { crypto } from "@ethersproject/crypto";
 const { Wallet } = require("../../dist/src/ethers"); // require("@klaytn/sdk-ethers");
```
Comment (kjeom) [inline ethers-ext/example/transactions/Basic_20_TxTypeAccountUpdate.js:2]: I mean it needs to be replaced to `require("@klaytn/ethers-ext")`

File: ethers-ext/example/transactions/Basic_20_TxTypeAccountUpdate.js (hunk 1 above)
Comment (nohkwak) [reply ethers-ext/example/transactions/Basic_20_TxTypeAccountUpdate.js:29]: Good comment. 
I will use original function ethers.utils.computePublicKey.

File: ethers-ext/example/transactions/Basic_20_TxTypeAccountUpdate.js (hunk 2 above)
Comment (nohkwak) [reply ethers-ext/example/transactions/Basic_20_TxTypeAccountUpdate.js]: Thanks.

**Review by kjeom — APPROVED** [review]

**Pull request merged**